README.md

```diff
@@ -1,81 +1,75 @@
-# Ruby Builder for Heroku
+# Ruby + JRuby Builder for Heroku
 
 This repo contains scripts to build binaries locally and on GitHub Actions.
 
 ## Building with GitHub actions
 
-Navigate to GithHub actions. Select the workflow "Build and upload Ruby runtime" then click the drop down "Run workflow" and enter the desired Ruby version. This will trigger a build for all supported stacks. If a version is not supported on a specific stack, add that logic to the `build.rb` file and to the GitHub action yaml logic.
+Navigate to GithHub actions. Select the workflow "Build and upload (j?)Ruby runtime" then click the drop down "Run workflow" and enter the desired Ruby version. This will trigger a build for all supported stacks. If a version is not supported on a specific stack, add that logic to the `inside_docker/*.rs` file and to the GitHub action yaml logic.
```

Comment:
Linking out to the workflow in the text ""Build and upload (j?)Ruby runtime" would be nice.